dist/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Restaurant</title>
+<link href="main.css" rel="stylesheet"></head>
+
+<body>
+  <div id="hero" class="w-screen h-screen bg-cover bg-bottom font-varela flex flex-col min-h-screen">
+
+    <nav class="px-8 pt-2 shadow-md">
+      <div class="-mb-px flex justify-center">
+        <a class="tab no-underline text-gray-200 border-b-2 border-transparent uppercase tracking-wide font-bold text-sm py-2 mr-8"
+          href="#" id="home">
+          Home
+        </a>
+        <a class="tab no-underline text-gray-200 border-b-2 border-transparent uppercase tracking-wide font-bold text-sm py-2 mr-8"
+          href="#" id="menu">
+          Menu
+        </a>
+        <a class="tab no-underline text-gray-200 border-b-2 border-transparent uppercase tracking-wide font-bold text-sm py-2"
+          href="#" id="contact">
+          Contact
+        </a>
+      </div>
+    </nav>
+
+    <main id="content" class="mx-auto w-5/6 p-2 m-8 shadow-md flex flex-grow h-112 items-center rounded-sm">
+    </main>
+
+    <footer>
+      <div class="flex justify-center p-2">
+        <a target="_blank" href="https://github.com/mosaaleb" class="text-gray-200 hover:underline">
+          Credits <span class="font-bold mx-2">|</span>
+        </a>
+        <a target="_blank" href="#" class="text-gray-200 hover:underline">
+          Reservation <span class="font-bold mx-2">|</span>
+        </a>
+        <a target="_blank" href="#" class="text-gray-200 hover:underline">
+          Reviews
+        </a>
+      </div>
+    </footer>
+
+  </div>
+<script type="text/javascript" src="main.js"></script></body>
+
+</html>
```

Comment:
The only change that is going to be required has to be done here. One of the requisites for the project to be 👍  as stated [HERE](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page)
![image](https://user-images.githubusercontent.com/46115725/71229878-eaf70d00-22ac-11ea-95af-0c5e175ab972.png)
This means that this particular file should only contain one div with the id of content, and the initial load of your site should be done with JS.
Just make sure you do that and you should be good to go.